source/slang/slang-ir-diff-jvp.cpp

```diff
@@ -474,20 +571,54 @@ struct JVPTranscriber
         case kIROp_Add:
         case kIROp_Mul:
         case kIROp_Sub:
+        case kIROp_Div:
             return differentiateBinaryArith(builder, instP);
 
         case kIROp_Construct:
             return differentiateConstruct(builder, instP);
         
         case kIROp_Call:
             return differentiateCall(builder, as<IRCall>(instP));
-
-        default:
-            getSink()->diagnose(instP->sourceLoc,
+        
+        case kIROp_swizzle:
+            return differentiateSwizzle(builder, as<IRSwizzle>(instP));
+        
+        case kIROp_constructVectorFromScalar:
+            return differentiateByPassthrough(builder, instP);
+        
+        case kIROp_Specialize:
+            // The implementation is still incomplete, but we sometimes encounter 
+            // *unused* specialize instructions. So, we're only going to handle that case.
+            if (!oldInstP->hasUses())
```

Comment:
Our DCE pass should be able to clear out these insts. I wonder if it is because specialize isn't marked as side-effect-free.